classes/LocalizationPack.php

```diff
@@ -109,7 +109,7 @@ public function loadLocalisationPack($pack, $selection, $install_mode = false, $
                 $id_lang = 1;
             }
 
-            if (!Language::isInstalled(Language::getIsoById($id_lang))) {
+            if (!empty($id_lang) && !Language::isInstalled(Language::getIsoById($id_lang))) {
```

Comment:
nothing to do with this PR but L102, do we really wanted to do an assignation ?

Reply:
Yes, we assign and check it just after as `Language::getIdByIso` can return 0.